rpm/SPECS/openresty-asan.spec

```diff
@@ -1,6 +1,6 @@
 Name:           openresty-asan
-Version:        1.15.8.2
-Release:        3%{?dist}
+Version:        1.17.1.1
```

Comment:
1.17.1.1 is not released yet. Also this change is unrelated to the current PR's title.

Reply:
Very well, I'll fork it out.